src/components/layers-panel/arcgis-import-panel/arcgis-import-panel.tsx

```diff
@@ -0,0 +1,220 @@
+import styled, { css, useTheme } from "styled-components";
+import { RadioButton } from "../../radio-button/radio-button";
+import { Fragment, useEffect, useState } from "react";
+import { PanelHorizontalLine } from "../../common";
+import SortDownIcon from "../../../../public/icons/arrow-down.svg";
+import { useAppDispatch, useAppSelector } from "../../../redux/hooks";
+import { ModalDialog } from "../../modal-dialog/modal-dialog";
+import {
+  selectArcGisContent,
+  selectArcGisContentSelected,
+  setArcGisContentSelected,
+  resetArcGisContentSelected,
+  setSortOrder,
+} from "../../../redux/slices/arcgis-content-slice";
+
+type InsertLayerProps = {
+  onImport: (object: { name: string; url: string; token?: string }) => void;
+  onCancel: () => void;
+};
+
+const Table = styled.div`
```

Comment:
Why divs? There are table, tr, th, td, thead, tbody in HTML

Reply:
I don't see a reason to re-work it. The current implementation works fine.

Reply:
It is important for code quality and from semantic markup perspective. We need to have it fixed.